.github/workflows/unit-tests.yml

```diff
@@ -17,7 +17,8 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: 1.17
+          go-version: 1.18
+          check-latest: true
```

Comment:
Along with using check-latest, I think we might need to specify the `go-version` with a caret to indicate 1.19.0 > version >= 1.18.0.

See the [supported version syntax for setup-go](https://github.com/actions/setup-go#supported-version-syntax)

Reply:
Never mind, I went further down the rabbithole to [npm/node-semver](https://github.com/npm/node-semver#x-ranges-12x-1x-12-) and 1.18 should be treated as 1.18.x.